images/omeka_classic/Dockerfile

```diff
@@ -45,6 +45,8 @@ RUN chown www-data:www-data application/config/config.ini
 
 RUN rm -rf /var/www/html/themes/* && rm /var/www/html/db.ini
 
+RUN rm -rf /var/www/html/plugins/ExhibitBuilder 
```

Comment:
@kelynch Is this to clear out a plugin that already exists in this version of Omeka and replace it with our version?

Reply:
@cole that's it!